.prettierrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  bracketSpacing: false,
```

Comment:
Pierdoła, ale mi sie latwiej czyta importy, kiedy sa te spacje w importach. Ale nie wazne, wazne, zeby wszedzie bylo tak samo.

Reply:
OK, no problem, dla mnie to jest sensowny argument, więc zmienię, jeśli nikomu nie przeszkadzają te spacje.

Reply:
Ja też wolę spacje w importach :)